cspell.json

```diff
@@ -90,6 +90,7 @@
     "tempdir",
     "testfn",
     "tikv",
+    "trival",
```

Comment:
This looks like `trivial`

Reply:
Is it my typo? Sorry 🤣

Reply:
😢 It my typo and misunderstand
I fixed it